model/encoder_decoder.py

```diff
@@ -60,11 +60,12 @@ def __init__(
         feature_info: FeatureInfo,
         metadata: Tuple[List[str], List[Tuple[str]]],
         embedding: bool,
+        heterogeneous_prop_agg_type: str, # "sum", "mean", "min", "max", "mul"
     ):
         super().__init__()
         self.embedding: bool = embedding
         self.encoder = GNNEncoder(encoder_layers)
-        self.encoder = to_hetero(self.encoder, metadata, aggr="sum")
+        self.encoder = to_hetero(self.encoder, metadata, aggr=heterogeneous_prop_agg_type)
```

Comment:
This is when the two seperate node types get aggregated?

Reply:
yeah... and they have different kind of options to conv layer agg_types...